specification/netapp/resource-manager/Microsoft.NetApp/stable/2020-11-01/netapp.json

```diff
@@ -0,0 +1,4873 @@
+          "type": "string"
+        }
+      }
+    },
+    "ResourceNameAvailabilityRequest": {
+      "description": "Resource name availability request content.",
+      "required": [
+        "name",
+        "type",
+        "resourceGroup"
+      ],
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Resource name to verify.",
+          "type": "string"
+        },
+        "type": {
+          "description": "Resource type used for verification.",
+          "enum": [
+            "Microsoft.NetApp/netAppAccounts",
+            "Microsoft.NetApp/netAppAccounts/capacityPools",
+            "Microsoft.NetApp/netAppAccounts/capacityPools/volumes",
+            "Microsoft.NetApp/netAppAccounts/capacityPools/volumes/snapshots"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "CheckNameResourceTypes",
+            "modelAsString": true
+          }
+        },
+        "resourceGroup": {
+          "description": "Resource group name.",
+          "type": "string"
+        }
+      }
+    },
+    "QuotaAvailabilityRequest": {
+      "description": "Quota availability request content.",
+      "required": [
+        "name",
+        "type",
+        "resourceGroup"
+      ],
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Name of the resource to verify.",
+          "type": "string"
+        },
+        "type": {
+          "description": "Resource type used for verification.",
+          "enum": [
+            "Microsoft.NetApp/netAppAccounts",
+            "Microsoft.NetApp/netAppAccounts/capacityPools",
+            "Microsoft.NetApp/netAppAccounts/capacityPools/volumes",
+            "Microsoft.NetApp/netAppAccounts/capacityPools/volumes/snapshots"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "CheckQuotaNameResourceTypes",
+            "modelAsString": true
+          }
+        },
+        "resourceGroup": {
+          "description": "Resource group name.",
+          "type": "string"
+        }
+      }
+    },
+    "netAppAccountList": {
+      "description": "List of NetApp account resources",
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "Multiple NetApp accounts",
+          "items": {
+            "$ref": "#/definitions/netAppAccount"
+          }
+        }
+      }
+    },
+    "netAppAccount": {
+      "description": "NetApp account resource",
+      "type": "object",
+      "x-ms-azure-resource": true,
+      "required": [
+        "location"
+      ],
+      "properties": {
+        "location": {
```

Comment:
In a new api version, Azure resource should support a top level property `systemData`.

https://armwiki.azurewebsites.net/api_contracts/ResourceSystemData.html

You can reference the one in common-type.
https://github.com/Azure/azure-rest-api-specs/blob/a16488bbd0d17551c729e22ea6ac6f7c6588a0a3/specification/common-types/resource-management/v2/types.json#L445

Reply:
Is a update to the swagger enough to support this or does hit require change on the RP service side or this is handeled by ARM?

Reply:
It requires changes in the RP. The above link has details.

It is okay to update the swagger without the RP changes, but the RP should implement the feature before releasing the SDKs to public.

Reply:
@j5lim  FYI the current dev version is 2021.02.01 and this backend change can be added in that but in the pipeline and scheduled for release are API versions 2020.11 and 2020.12.  We need to add those swagger API versions for Completeness. 
As this backend update may take some time to get implemented and released so we ask if is this change is strictly mandatory and so urgent that it blocks this update or if can we finalize the implementation in the backend and add in this in the next upcoming 2021.02 that is scheduled soon.

Reply:
@j5lim  can you comment on this.

Reply:
@j5lim @yungezz  can you comment on this?